### ISSUE TITLE
Accessibility - A tag missing descriptive text

### ISSUE THREAD

**Issue by axamei** [pulibrary/special_collections#394]
The accessible name of a link must accurately describe the target or purpose of the link. 

Add text content to the 5 empty links that describes the target of each link. Menu grid: conducting research and ask a question; New and notable: Images.
